src/NLog/project.json

```diff
@@ -78,27 +78,58 @@
     //    "define": [ "NET4_0", "WCF_SUPPORTED", "NETSTANDARD1_0", "NETSTANDARD" ]
     //  }
     //},
+    //"uap": {
```

Comment:
You almost have it. I had UWP building with my `project.json` configuration [here](https://github.com/NLog/NLog/pull/2053/files/bf377d08a97447ff5f94a6961a535570dffa554d#diff-4c95cd96a32a135a3f845aad76498b78R226). I think `uap` needs to be `uap10.0` and `NETStandard.Library` replaced with `Microsoft.NETCore.UniversalWindowsPlatform`.